plottr/data/datadict.py

```diff
@@ -1067,7 +1080,11 @@ def guess_shape_from_datadict(data: DataDict) -> \
     shapes = {}
     for d in data.dependents():
         axnames = data.axes(d)
-        axes = {a: data.data_vals(a) for a in axnames}
+        axes: Dict[str, np.ndarray] = {}
+        for a in axnames:
+            axdata = data.data_vals(a)
+            assert isinstance(axdata, np.ndarray)
```

Comment:
do i understand corectly that this won't be needed once `def data_vals(self, key: str) -> Union[Sequence, np.ndarray]:` is reworked/retyped to always return numpy arrays? btw, can it? (perhaps also a question to @wpfff )

Reply:
Yes indeed there are a lot of work arounds relateing to that union that could be removed if datavals was always np.ndarray.